textx/export.py

```diff
@@ -103,6 +103,14 @@ def dot_escape(s):
             .replace('?', r'\?')
 
 
+def html_escape(s):
+    if sys.version < '3':
+        return s.replace("<", "&lt;").replace(">", "&gt;")
+    else:
+        from html import escape
+        return escape(s)
+
```

Comment:
This is a minor and won't make much difference in this case but it is usually a good thing to do something like:
```python
if sys.version < '3':
    def html_escape(s):
        return s.replace("<", "&lt;").replace(">", "&gt;")
else:
    def html_escape(s):
        from html import escape
        return escape(s)
```

As the condition will be evaluated only once on module import, not every time the func is called.

Reply:
done